llvm/test/CodeGen/AMDGPU/GlobalISel/ubsan_trap.ll

```diff
@@ -0,0 +1,13 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -global-isel < %s | FileCheck %s
```

Comment:
This should probably just merge in with trap-abis.ll and/or trap.ll. You're not testing both dag (which looks untested) or the various ABI options